pkgs/by-name/at/atlauncher/package.nix

```diff
@@ -39,22 +38,22 @@ stdenv.mkDerivation (finalAttrs: {
 
   desktopItems = [
     (makeDesktopItem {
-      name = finalAttrs.pname;
-      exec = finalAttrs.pname;
-      icon = finalAttrs.pname;
-      desktopName = "ATLauncher";
       categories = [ "Game" ];
+      desktopName = "ATLauncher";
+      exec = "atlauncher";
+      icon = finalAttrs.pname;
+      name = finalAttrs.pname;
```

Comment:
Please don't use this.

`name = finalAttrs.pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Yeah, I've seen that issue. Didn't know it applied to more than just the binary output(s).

So, I change this for both `icon` and `name`, right?

Reply:
Yes, use literal strings. Thanks!